doc/tasks/git_commit_message.md

```diff
@@ -10,8 +10,12 @@ parameters:
         git_commit_message:
             allow_empty_message: false
             enforce_capitalized_subject: true
+            enforce_no_subject_punctuations: false
             enforce_no_subject_trailing_period: true
             enforce_single_lined_subject: true
+            enforce_type_scope_conventions: false
+            types: []
```

Comment:
I think it would be better to nest the options that are only used by `enforce_type_scope_conventions` to be nested under it. So that it is clear that they can only be used in conjunction with the `enforce_type_scope_conventions` option.

Reply:
I thought about this as well, i will fix this up and will push it up. Stand by ...